nss-day-cohort-52/giffygram-wild-hurricanes
# main.js and provider.js

#### Issues Completed
Applies to issue # n/a

#### Changes Made
1.  [] Feature
    [] Bug-fix
    [] Feature that will cause a bug
    [x] Not a feature or a bug- requires documentation
2. Added file `database.json` to `api` directory and removed example file
3. Modified file `provider.js` to include get and fetch functions. Modified main.js to include fetch imports and stateChanged customevent.
4. Make sure your code has been commented.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    commit your current code (git add --all, git commit -m "")
    git fetch --all
    git checkout branchname
    ```
2. Serve the code to browser.
3. Test app functionality.
    > How should the reviewer test functionality?
    > What is the intended outcome?
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
